src/example-styles.css

```diff
@@ -24,25 +24,11 @@
     white-space: nowrap;
 }
 
-.multi-select-menuitem--titled:before {
-    display: block;
-    font-weight: bold;
-    content: attr(data-group-title);
-    margin: 0 0 0.25em -20px;
-}
-
-.multi-select-menuitem--titledsr:before {
-    display: block;
+legend {
+    font-size: 0.875em;
     font-weight: bold;
-    content: attr(data-group-title);
-    border: 0;
-    clip: rect(0 0 0 0);
-    height: 1px;
-    margin: -1px;
-    overflow: hidden;
-    padding: 0;
-    position: absolute;
-    width: 1px;
+    padding-left: 10px;
+    padding-top: 0.25rem;
```

Comment:
Dunno if this is just me used to the old one, but this felt a bit spaced out to me now, especially if there's going to be a lot of options? What do you think about dropping this padding-top and adding a:
`.multi-select-legend + .multi-select-menuitem { padding-top: 0.25rem; }`
to reduce the spacing a bit?